Cannot run tests on clean checkout

When running tests from a fresh checkout (ie no dist/ dir present) the following error is thrown

```
Error: Cannot find module '../dist/wrapper'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/rtirserio/Developer/contentpull/test/main.spec.js:11:1)
...
```

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "cover": "nyc npm t && npm run coverage-report",
     "coverage-check": "nyc check-coverage --branches 100 --statements 100 --functions 100 --lines 100",
     "coverage-report": "nyc report --reporter=lcov",
-    "test": "_mocha --compilers js:babel-register",
+    "test": "npm run build && _mocha --compilers js:babel-register",
     "prebuild": "rimraf dist",
     "build": "babel --copy-files --out-dir dist src",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",

